web/beacon-app/src/features/auth/components/Register/RegistrationForm.tsx

```diff
@@ -224,19 +109,35 @@ function RegistrationForm({ onSubmit }: RegistrationFormProps) {
           className="mt-4"
           isLoading={isSubmitting}
           disabled={isSubmitting}
-          aria-label={t`Create Starter account`}
+          aria-label={t`Create Free Account`}
         >
-          <Trans>Create Starter Account</Trans>
+          <Trans>Create Free Account</Trans>
         </Button>
-        <p className="mt-2">
-          <Trans>No cost. No credit card required.</Trans>
+        <p className="mt-3">
+          <Trans>By continuing, you're agreeing to the Rotational Labs Inc.</Trans>{' '}
+          <Link
+            to={EXTERNAL_LINKS.TERMS}
+            className="font-bold text-[#1F4CED] underline"
+            target="_blank"
+          >
+            <Trans>Terms of Service</Trans>
+          </Link>{' '}
+          <Trans>and</Trans>{' '}
+          <Link
+            to={EXTERNAL_LINKS.PRIVACY}
+            className="font-bold text-[#1F4CED] underline"
+            target="_blank"
+          >
+            <Trans>Privacy Policy</Trans>
+          </Link>
+          .
         </p>
       </Form>
     </FormikProvider>
   );
 }
 
-const Fieldset = styled.fieldset`
+/* const Fieldset = styled.fieldset */ `
```

Comment:
I didn't delete the styles in case they may be needed again if changes are made. However, I can remove them if you think that'll be better.